src/components/Common/Typography/Text.tsx

```diff
@@ -10,7 +10,7 @@ const DEFAULT_FONT_SIZE: FontSize = 'md'
 const DEFAULT_FONT_STYLE: FontStyle = 'normal'
 export type FontSize = 'xs' | 'sm' | 'md' | 'lg' | 'xl'
 export type FontWeight = 'bold' | 'semi-bold' | 'normal' | 'medium' | 'light'
-type TextColor = 'default' | 'primary' | 'secondary' | 'error'
+type TextColor = 'default' | 'primary' | 'secondary' | 'error' | 'white'
```

Comment:
Maybe, as a pattern, we could use `white-900` here so it is strictly related to the color🤔